Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al

```diff
@@ -26,6 +26,35 @@ codeunit 20352 "Connectivity App Definitions"
         RegisterWiseBanking();
         RegisterDirectBankingNL();
         RegisterSofteraBankfeed();
+        RegisterSUManGOAutoBank();
+    end;
+    local procedure RegisterSUManGOAutoBank()
+    var
+        AppId: Text[250];
+        AppName: Text[1024];
+        AppPublisher: Text[250];
+        AppDescription: Text[2048];
+        AppProviderSupportURL: Text[250];
+        AppSourceURL: Text[250];
+        AppWorksOn: Text;
+        AppApprovedFor: Text;
+    begin
+        /***************************************************
+            Add app 'Sumango AutoBank' to NO, SE
+        ***************************************************/
+
+        AppId := '52508c8d-fb2b-49da-9657-f49859fd43cc';
+        AppName := 'Sumango AutoBank';
+        AppPublisher := 'Sumango AS';
+        AppDescription := 'AutoBank manage payments, recivables, refunds and reconciliation in all currencies directly from Business Central. Streamlines, simplifies and expands bank functionality without leaving your Business Central environment. Sumango AutoBank support banks in Norway, Sweden, Denmark, Finland, Germany, Portugal, United Kingdom and United States. Visit www.sumango.no for more information.';
+        AppProviderSupportURL := 'https://www.sumango.no';
```

Comment:
This URL needs to point to a page where the supported banks are listed. It seems like it would be this one?
https://oseberg.atlassian.net/wiki/spaces/Autobank/pages/2342191252/Integrated+banks

Reply:
Yes, should I make an update or can you replace?

Reply:
To ensure proper process, it is best if you change. Thank you.